core/src/main/java/io/grpc/ClientCall.java

```diff
@@ -252,4 +252,14 @@ public boolean isReady() {
   public void setMessageCompression(boolean enabled) {
     // noop
   }
+
+  /**
+   * Returns a set of attributes, which may vary depending on the particular
+   * implementation and the state of the call, channel, or transport at the moment it is being
+   * called. By default it returns EMPTY attributes.
+   */
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/2607")
+  public Attributes getAttributes() {
```

Comment:
Can we please make it consistent with the `ServerCall` and rename it to `attributes()` ? https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/ServerCall.java#L230-L242